group_vars/all.yml

```diff
@@ -287,18 +287,27 @@ nfs_exports:
 # Cloudflare is a great free DNS option for domains. If you use the cloudflare_ddns container then you'll need to
 # set the options below.
 
-# Your domain name
+# Your domain name (in the free subscription this is always your TLD such as example.com)
 cloudflare_zone: "{{ ansible_nas_domain }}"
 
-# The hostname you want the container to update. You shouldn't need to change this.
-cloudflare_host: "*.{{ cloudflare_zone }}"
+# The hostname/subdomain you want the container to update.
+# To register a hostname use "{{ ansible_nas_hostname }}.{{ cloudflare_zone }}"
+# To register a subdomain use "*.<subdomain>.{{ cloudflare_zone }}"
+# To forward any use "*.{{ cloudflare_zone }}"
+cloudflare_subdomain: "*.{{ cloudflare_zone }}"
```

Comment:
I think this should just be "*", as it resulted in "*.{{ cloudflare_zone }}.{{ cloudflare_zone }}" for me

Reply:
How did you define cloudflare_zone?
Basically it is defined by the following parameters:

```
ansible_nas_hostname: "gatehost"
ansible_nas_domain: "example.com"
cloudflare_zone: "{{ ansible_nas_domain }}"
cloudflare_subdomain: "*.{{ cloudflare_zone }}"
```

which will result in `cloudflare_subdomain="*.example.com"`.

For my setup I use `cloudflare_subdomain: "{{ ansible_nas_hostname }}.{{ cloudflare_zone }}"` which results in `gatehost.example.com`.